src/jagadeeshMJun20/FirstAndLastIndex.java

```diff
@@ -0,0 +1,26 @@
+package jagadeeshMJun20;
+
+public class FirstAndLastIndex {
+		void printCharFreq(String name) {
+			for(int index=0; index<name.length(); index++) {
+				char ch = name.charAt(index);//t
+				int count = 0;
+				if (index == name.indexOf(ch)) {
+				     for(int i=0; i<name.length(); i++) {
+					 if (ch == name.charAt(i)) 
+						count++;
+						 if (count>1) {
+								int num = name.indexOf(ch);
```

Comment:
You have increased the complexity in code, Using one for loop and one if block you can achieve the output. Think once !!